code/__HELPERS/unsorted.dm

```diff
@@ -833,7 +833,7 @@ GLOBAL_LIST_INIT(common_tools, typecacheof(list(
 
 
 /proc/params2turf(scr_loc, turf/origin, client/C)
-	if(!scr_loc)
+	if(!scr_loc || !origin)
```

Comment:
Where is this being called without passing the origin?

Reply:
The description explains it.
Click on a null object (like a splash screen before init), since it's null it will have a null turf